src/app/pages/landing/landing.component.html

```diff
@@ -16,5 +16,10 @@ <h1 class="logo">Mentor Me</h1>
       <p class="sub-text">A Value exchange 
         application</p>
     </ion-text>
+    <div class="buttons">
+      <ion-button class="btn select-role-btn mentee-btn">Mentee</ion-button>
```

Comment:
This should be a toggle component looks like button but is actually a radio buttons. You can delete this from here and commit then on another story work on the component and bring into this page